<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlannerMonitor</class>
 <widget class="QMainWindow" name="PlannerMonitor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1354</width>
    <height>769</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>PlannerMonitor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QwtPlot" name="plot2Dworld">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>10</y>
      <width>650</width>
      <height>670</height>
     </rect>
    </property>
    <property name="propertiesDocument" stdset="0">
     <string/>
    </property>
   </widget>
   <widget class="QTabWidget" name="myGLWidget">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>10</y>
      <width>661</width>
      <height>601</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Simulator</string>
     </attribute>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>10</y>
        <width>31</width>
        <height>541</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_30">
      <property name="geometry">
       <rect>
        <x>660</x>
        <y>10</y>
        <width>71</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>CONTROL</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>210</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>History</string>
      </property>
     </widget>
     <widget class="QwtPlot" name="plotHistoryCurvature">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>230</y>
        <width>411</width>
        <height>161</height>
       </rect>
      </property>
     </widget>
     <widget class="QwtPlot" name="plotHistorySpeed">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>410</y>
        <width>411</width>
        <height>151</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnLoadScenarioFromFile">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>121</width>
        <height>51</height>
       </rect>
      </property>
      <property name="text">
       <string>Load Scenario</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabLocalization">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <attribute name="title">
      <string>Map / Localization Input</string>
     </attribute>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>10</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Vehicle State @ C.G.</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>51</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Easting</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>50</y>
        <width>61</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Northing</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>70</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Yaw</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>90</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Curv</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineVsEasting">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>30</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineVsNorthing">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>50</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineVsYaw">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>70</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineVsCurv">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>90</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnSetSimVs">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>120</y>
        <width>121</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Set To Sim</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>41</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Speed</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineVsSpeed">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>110</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_12">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>31</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Acc</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineVsAcc">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>130</y>
        <width>101</width>
        <height>21</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="lblVsEasting">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblVsNorthing">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>50</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblVsYaw">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>70</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblVsCurv">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>90</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblVsSpeed">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>110</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblVsAcc">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>130</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>0.0</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnUpdateFromSimVs">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>30</y>
        <width>121</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>Update From Sim</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>170</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Obj_Static</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_18">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>170</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Obj_Pedestrian</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_28">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>170</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Obj_Bicyclist</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_29">
      <property name="geometry">
       <rect>
        <x>360</x>
        <y>170</y>
        <width>131</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Obj_Vehicle</string>
      </property>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>10</y>
        <width>31</width>
        <height>541</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
     <widget class="QLabel" name="label_31">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>10</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>DISPLAY</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnToggleDisplayLocalization">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>230</y>
        <width>31</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>OFF</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_16">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>230</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Vehicle State</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_19">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>310</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Perception</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>270</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Map Waypoints</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btnToggleDisplayMap">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>270</y>
        <width>31</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>OFF</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnToggleDisplayPerception">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>310</y>
        <width>31</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>OFF</string>
      </property>
     </widget>
     <widget class="MyGLWidget" name="openGLWidget">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>220</y>
        <width>331</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnToggleDisplaySim_GroundTruthObjects">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>120</y>
        <width>31</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>OFF</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_39">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>120</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Ground Tr. Object</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>40</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Road Structure</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QLabel" name="label_32">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>80</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Vehicle Pose</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
      </property>
     </widget>
     <widget class="QPushButton" name="btnToggleDisplaySim_VehiclePose">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>80</y>
        <width>31</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>OFF</string>
      </property>
     </widget>
     <widget class="QPushButton" name="btnToggleDisplaySim_RoadStructure">
      <property name="geometry">
       <rect>
        <x>620</x>
        <y>40</y>
        <width>31</width>
        <height>27</height>
       </rect>
      </property>
      <property name="text">
       <string>OFF</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Steering Input</string>
     </attribute>
     <widget class="QwtPlot" name="plotSteeringWheel">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>40</y>
        <width>331</width>
        <height>231</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnJoystickOverride">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>500</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>JS Switch</string>
      </property>
     </widget>
     <widget class="QLabel" name="lblJSSwitch">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>470</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>KacstLetter</family>
        <pointsize>15</pointsize>
        <underline>false</underline>
        <strikeout>false</strikeout>
        <kerning>true</kerning>
       </font>
      </property>
      <property name="text">
       <string>IDLE</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QPushButton" name="btnSimSwitch">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>650</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Sim Switch</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbSimSwitch">
    <property name="geometry">
     <rect>
      <x>930</x>
      <y>620</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>KacstLetter</family>
      <pointsize>15</pointsize>
      <underline>false</underline>
      <strikeout>false</strikeout>
      <kerning>true</kerning>
     </font>
    </property>
    <property name="text">
     <string>Unlinked</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnWorldZoomIn">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>620</y>
      <width>87</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom In</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnWorldZoomOut">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>660</y>
      <width>87</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Zoom Out</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnWorldUp">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>620</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Up</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnWorldDown">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>660</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Down</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnWorldLeft">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>640</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Left</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnRight">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>640</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Right</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="chkSimIsolated">
    <property name="geometry">
     <rect>
      <x>1030</x>
      <y>650</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Sim Isolated</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1354</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QFrame</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MyGLWidget</class>
   <extends>QOpenGLWidget</extends>
   <header>vis/myglwidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
